src/builtin_plugins.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import Plugins, {Plugin} from './plugins'
+import path from 'path'
+
+export default class BuiltinPlugins {
+  constructor (plugins: Plugins) {
+    this.plugins = plugins
+  }
+
+  plugins: Plugins
+
+  /**
+   * list user plugins
+   * @returns {Plugin[]}
+   */
+  list (): Plugin[] {
+    let commandsPath = path.resolve(path.join(__dirname, 'commands'))
```

Comment:
also, any particular reason for using `path.resolve` here? My understanding is it is the same unless you're using relative paths, so it should be fine